express/component_spec.py

```diff
@@ -16,29 +16,23 @@
 from express.schema import Field
 
 
-def get_kubeflow_type(python_type: str):
-    """
-    Function that returns a Kubeflow equivalent data type from a Python data type
-    Args:
-        python_type (str): the string representation of the data type
-    Returns:
-        The kubeflow data type
-    """
-    mapping = {
-        "str": "String",
-        "int": "Integer",
-        "float": "Float",
-        "bool": "Boolean",
-        "dict": "Map",
-        "list": "List",
-        "tuple": "List",
-        "set": "Set",
-    }
-
-    try:
-        return mapping[python_type]
-    except KeyError:
-        raise ValueError(f"Invalid Python data type: {python_type}")
+python2kubeflow_type = {
+    "str": "String",
+    "int": "Integer",
+    "float": "Float",
+    "bool": "Boolean",
+    "dict": "Map",
+    "list": "List",
+    "tuple": "List",
+    "set": "Set",
+}
+
+kubeflow2python_type = {
```

Comment:
Some datatypes are missing, Can't we just invert the one from above?

Reply:
I think it's fine for now since we're not using any other data types or the current testing pipeline. This is currently used to convert the data type in the argument parser but won't be needed when upgrading to kfpv2. 

Will add a TODO.